libraries/abstractions/common_io/test/iot_test_common_io.c

```diff
@@ -57,6 +57,14 @@ TEST_GROUP_RUNNER( Common_IO )
     size_t i = 0;
 
     /* These already used loop back tests which require minimum of two pins */
```

Comment:
This comment only applies to GPIO tests, Should go inside or above it's `#ifdef IOT_TEST_COMMON_IO_GPIO_SUPPORTED` block